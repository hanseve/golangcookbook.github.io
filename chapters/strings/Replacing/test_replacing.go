package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println(strings.Replace("oink oink oink", "k", "ky", 2))
	fmt.Println(strings.Replace("oink oink oink", "oink", "moo", -1))

	fmt.Println(strings.ReplaceAll("oink oink oink", "oink", "moo"))
}
