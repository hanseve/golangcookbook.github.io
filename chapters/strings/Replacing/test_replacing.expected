oinky oinky oink
moo moo moo
moo moo moo
